instructors.md

```diff
@@ -529,36 +551,35 @@ Solutions to exercises:
 >         assert result[-1] >= result[0]
 >     return result
 > ~~~
-> ~~~ {.output}
-> The first assertion checks that the input sequence `values` is not empty.
->   An empty sequence such as `[]` will make it fail.
-> The second assertion checks that the first value in the list is positive.
->   Input such as `[-1,0,2,3]` will make it fail.
-> The third assertion checks that the running total always increases.
->   Input such as `[0,1,3,-5,4]` will make it fail.
-> ~~~
+>
```

Comment:
I think my preference would probably be to remove the question, just to reduce duplication, as when we make minor changes to the questions we're unlikely to update them here as well. Mainly, I changed it because the `{.output}` also makes it a `pre` block, so things like the backticks here don't make a difference, and everything is rendered in a monospaced font.

I think ideally, we would implement a `{.answer}` style, which could even hide the answer until a click or something like that. For the time being, I'll add in a horizontal rule to at least give some visual clue that the answer's started.